core/src/main/java/org/apache/iceberg/actions/BaseRewriteDataFilesAction.java

```diff
@@ -65,6 +65,7 @@
   private Expression filter;
   private long targetSizeInBytes;
   private int splitLookback;
+  private int itemsPerBin;
```

Comment:
Looks like this is only modifying the old rewrite path and not the new path. We should probably fix it in both places

Reply:
Well, what new path you mean, `RewriteDataFiles` and related implements?

Reply:
Yeah the new api extends off the interface rather than this base class.